### ISSUE TITLE
OS-Build is having some cache issues

### ISSUE THREAD

**Issue by sidpremkumar** [sidpremkumar/Sync2Jira#96]
Something is up with openshift build, I believe it has to do with the python cached modules. 

I am proposing to install virtualenv and use that when running the tests so each one is fresh.

**Comment by sidpremkumar** [comment]
So it turns our that when we installed Sync2Jira in `Dockerfile.build` was causing import issues when we had `from sync2jira.main import main` in our `integration_test.py`. 

Instead of pip installing, we needed to `python3 setup.py develop` so that we could register changes

The current branch of `openshift-build` is now working, and finalizing some finishing touches befor eI put up a PR. 

We now run clone the Sync2Jira repo into `openshift-build/temp` and then run `python3 setup.py. develop` in our `Dockerfile.build`. We then run something like this: 
```
git fetch --all
git reset --hard origin/${BRANCH}
```
When we want to switch branches to run tests.
